src/Log/Engine/FileLog.php

```diff
@@ -116,7 +116,6 @@ public function __construct(array $config = [])
      */
     public function log($level, $message, array $context = []): void
     {
-        $message = $this->_format($message, $context);
```

Comment:
Shouldn't we still call format() to enable subclasses to do their own formatting?

Reply:
Yup this statement was mistakenly removed, reverted.